CHANGELOG.md

```diff
@@ -1,9 +1,7 @@
-# Changelog
+# Change Log
 
 All notable changes to this project will be documented in this file.
-
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
Are we changing the format because of lerna?

Reply:
Yes